bin/publish

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Shellcheck? I just ran it locally and nothing was reported back. If we are introducing scripting do we need to pull along the `linting` for it? Probably overkill but I think it's worth asking. 

If we are starting to do scripting do we want to pull this into codeship? Again, probably overkill but worth asking about. I think knowing that the deployment is scripted would make me more likely to want to make a quick tweak to the rules.

Reply:
I think this problem will be easy to solve if we switch to GitHub actions. However, for this particular use case, I don't feel like it's worth it. I ran it locally while I wrote the script.